gimp/pictonode-gimp-plugin/GIMP 2.99/pictonode/custom_nodes.py

```diff
@@ -132,6 +132,9 @@ def change_layer(self, combo):
     def remove(self, node):
         self.destroy()
 
+    def get_values(self):
+        return {}
```

Comment:
(`ImgSrcNode`) This should have a custom value that expresses either a file name or a file path.